dll/appcompat/apphelp/apphelp.c

```diff
@@ -185,7 +185,7 @@ ApphelpCheckRunAppEx(
     _In_ HANDLE FileHandle,
     _In_opt_ PVOID Unk1,
     _In_opt_ PVOID Unk2,
-    _In_opt_z_ PWCHAR ApplicationName,
+    _In_opt_z_ LPCWSTR ApplicationName,
```

Comment:
PCWSTR if the code is NT-oriented. (Otherwise you would have use LPVOID, WORD, etc. for the other types.)